src/LIBRETRO/libretro.c

```diff
@@ -291,7 +292,7 @@ static void handle_input(void)
       handle_key(KEY88_A + i, RETROK_a + i);
 
    /* Function keys */
-   for (i = 0; i < 8; i++)
+   for (i = 0; i < 5; i++)
```

Comment:
There are only 5 function keys mapped starting at KEY88_F6: https://github.com/libretro/quasi88-libretro/blob/ff94d9d2884d71580c163fcd71b6850c0f393f44/src/keyboard.h#L342-L352

This was causing F6 to be mapped to KEY88_BS, F7 to KEY88_INS and F8 to KEY88_DEL.

Reply:
Combined with not checking the key state above, these were the only ways to press BS, INS, and DEL.